client/src/containers/ClubProfile.js

```diff
@@ -0,0 +1,264 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from 'react-router-dom';
+import styled from 'styled-components';
+import ExploreBox from '../components/explorebox/index';
+import MainContent from '../components/profileMainContent/index';
+import { ProfilePageBox } from "../components/profilePageBox";
+import { Navbar } from "../components/navbar";
+import { FrequencyTag } from "../components/frequencyTag/index";
+
+export const ClubProfile = () => {
+    const { id } = useParams();
+    const [club, setClub] = useState();
+    const [width, setWidth] = useState(window.innerWidth);
+    const [isLoading, setLoading] = useState(true);
+    const options = { year: 'numeric', month: 'long', day: 'numeric' };
+
+    useEffect(() => {
+        window.addEventListener("resize", () => setWidth(window.innerWidth));
+        fetch(`api/clubProfiles/${id}`, {
+            method: 'GET',
+            })
+          .then((res) => res.json())
+          .then((response) => {
+            setClub(response);
+            setLoading(false);
+            console.log(`api/clubProfiles/${id}`);
+          })
+          .catch((error) => console.log(error));
+    }, [id]);
+    
```

Comment:
I would recommend adding this variable and then replacing all instances of the similar clubs code with `<SimilarClubs />` in order to make the code shorter/more readable
```suggestion
        const SimilarClubs = () => { return (club.similarClubs.length === 0) ? (<h1>Loading</h1>) : 
        (club.similarClubs.map(profile => (
            <ExploreBox 
                name = {profile.name}
                description = {profile.shortDescription}
                imageURL = {profile.imageUrl}
                tags = {profile.tags}
                clubSize = {profile.memberRange}
                acceptingMembers = {profile.acceptingMembers}
                applicationRequired = {profile.applicationRequired}
                cardLink={`/clubs/${profile._id}`}
            />
    )))};
```